core/themes/basis/css/component/hero.css

```diff
@@ -94,3 +94,21 @@
     font-size: 2.625em;
   }
 }
+
+/**
+ * Some changes to fix problems with extra white space between blocks in Basis.
+ * Is there a better location?
+ **/
+
+.update-1-30 .block {
```

Comment:
this changes all blocks. Is that really what you wanted?

Reply:
@jenlampton Yes, I explained this the main issue. I will go ahead and change the issue title. Because, I think it's very important to fix this for more than just hero blocks.